snyk-monitor/templates/deployment.yaml

```diff
@@ -20,6 +20,13 @@ spec:
     spec:
       serviceAccountName: {{ include "snyk-monitor.name" . }}
       restartPolicy: Always
+      initContainers:
+        - name: volume-permissions
+          image: busybox
+          command : ['sh', '-c', 'chmod -R 777 /var/tmp']
```

Comment:
777 is really lax, but nothing less restrictive seems to work, because we're not guaranteed to know the user/group id until everything is deployed. Can anyone suggest a better alternative?

Reply:
Do we need the `execute` bit? We usually only read and write files to that directory

Reply:
Good point. It has execute by default, but you're right that I dont think it's needed - will test and update 👍

Reply:
Just tested and even chmod 776 without  the `execute` permissions seems to bring the "Permission denied" error back - looks like it is needed for some reason 🤔